neutrino-custom/base.js

```diff
@@ -212,6 +230,7 @@ module.exports = neutrino => {
     neutrino.config
         .plugin('provide')
         .use(webpack.ProvidePlugin, {
+            auth0: require.resolve('auth0-js'),
```

Comment:
This list here is leftover from before we used webpack. We're moving towards using imports where possible - could this be imported instead?